src/main/webapp/i18n/de/exam.json

```diff
@@ -573,7 +576,8 @@
                     "examReview": "Alle relevanten Daten müssen in der Klausur spezifiziert sein.",
                     "resolveComplaints": "Beschwerden können im Bewertungs-Dashboard bearbeitet werden.",
                     "testRun": "Probiere verschiedene Kombinationen von Übungsvarianten aus Sicht der Studierenden aus.",
-                    "exportResults": "Die Klausurergebnisse können in Form einer .csv Datei exportiert werden."
+                    "exportResults": "Die Klausurergebnisse können in Form einer .csv Datei exportiert werden.",
+                    "suspiciousBehavior": "Verdächtiges Verhalten und Plagiatsfälle können im Verdächtiges Verhalten Dashboard eingesehen werden."
```

Comment:
Due to the name of the dashboard, this sentence seems off. Maybe formatting would do the trick but I'm not sure if we did this anywhere else ...

Reply:
rephrased the sentence.